coops-ui/tests/unit/Certify.spec.ts

```diff
@@ -0,0 +1,172 @@
+//
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+
+import { mount, Wrapper } from '@vue/test-utils'
+
+import Certify from '@/components/AnnualReport/Certify.vue'
+
+Vue.use(Vuetify)
+
+// Boilerplate to prevent the complaint "[Vuetify] Unable to locate target [data-app]"
```

Comment:
This boilerplate is not needed in every file. Did you check without it?

Reply:
I didn't check - works fine without it.